src/main/osd/osd_warnings.c

```diff
@@ -294,9 +298,10 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
                 *p++ = 'T';
                 escWarning = true;
             } else if (osdConfig()->esc_rpm_alarm != ESC_RPM_ALARM_OFF
+                       && motor[i] > mixerRuntime.disarmMotorOutput
```

Comment:
- This code is duplicated below
- is should be implemented as property of motor/mixer

Reply:
@coderabbitai please create a commit with these changes

Reply:
@coderabbitai you do not have to use my name for this. Just add another commit with your proposed changes on your own PR.